### ISSUE TITLE
Dataset Required. Label System

### ISSUE THREAD

**Issue by Micae1o** [Hamagistral/TeethSeg#2]
May I ask what kind of tooth labeling system you use? That is, what labels were used to label the tooth grids in the training data? What training data did you use to train the neural network?

In the code, I saw the number 15 for labels. But which specific teeth belong to which number?
